tests/cli/test_cli.py

```diff
@@ -560,6 +560,12 @@ def run(cls, tests, instance, second_instance):
 ############################################################
 
 
+@TestModule.register('unittest')
```

Comment:
@yiboliu you have to give the TestModule.register a different name, I think, for it to actually be run in the tests.